guides/common/assembly_managing-container-images.adoc

```diff
@@ -1,11 +1,4 @@
-[id="Managing_Container_Images_{context}"]
-= Managing Container Images
-
-With {Project}, you can import container images from various sources and distribute them to external containers using Content Views.
-
-ifndef::orcharhino[]
-For information about containers, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/getting_started_with_containers/index[Getting Started with Containers] in _Red Hat Enterprise Linux Atomic Host 7_.
-endif::[]
+include::modules/con_managing-container-images.adoc[]
```

Comment:
Mark -- see my comment for `assembly_synchronizing-content-between-foreman.adoc`